gammapy/utils/random/inverse_cdf.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from .utils import get_random_state
+
+
+class InverseCDFSampler:
+    """Inverse CDF sampler.
+       
+   It determines a set of random numbers and calculate the cumulative 
+   distribution function.
+   
+   Parameters
+   ----------
+   pdf : `~gammapy.maps.Map`
```

Comment:
Is `pdf` really ` Map` object?
Should be a Numpy array, no?

Also, it probably should be a probability mass function `pmf` (that's what it's called in scipy.stats), i.e. finite probability per bin, no?

Reply:
Yes, correct. It should be a generic Numpy array. I think Axel is going to modify the code soon.

In fact we are working with binned distribution, so the mass function should be a more adequate naming convention.